Tasks/JenkinsQueueJobV2/jenkinsqueuejobtask.ts

```diff
@@ -119,12 +122,18 @@ async function doWork() {
         const taskOptions: TaskOptions = new TaskOptions();
         const jobQueue: JobQueue = new JobQueue(taskOptions);
         const queueUri = await util.pollSubmitJob(taskOptions);
-        console.log(tl.loc('JenkinsJobQueued'));
-        const rootJob = await util.pollCreateRootJob(queueUri, jobQueue, taskOptions);
-        //start the job queue
-        jobQueue.Start();
-        //store the job name in the output variable
-        tl.setVariable('JENKINS_JOB_ID', rootJob.ExecutableNumber.toString());
+
+        if (queueUri == '302') {
```

Comment:
Why queueUri could be 302 here?

Reply:
It's from these 2 points in code: [1](https://github.com/denis-tikhomirov/azure-pipelines-tasks/blob/19610e2c2001da71d1ed01e15ceb6677a9b0f860/Tasks/JenkinsQueueJobV2/util.ts#L274) and [2](https://github.com/denis-tikhomirov/azure-pipelines-tasks/blob/19610e2c2001da71d1ed01e15ceb6677a9b0f860/Tasks/JenkinsQueueJobV2/util.ts#L284).
Usually string with URL to new started build must be presented here. But I'm not sure if URL of new started build is provided with returned code 302 (I don't have repro of the issue and customer didn't provide me details about repro). So I added assigning of string '302' to `queueUri` to identify code 302 as result of starting of build.

Reply:
I moved handling of code 302 to catch statement in jenkinsqueuejobtask.ts file